### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Lawmsangi**
In this project, I implemented the following:

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- Add 2 routes in your application, each should render a page component:
- [ ] - /
- [ ] - /categories

- Create components inside of the /components directory:
- Contains individual book state, as well as a button to delete the book
- Contains a list to renders individual books
- Contains a form to create new books
- Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,128 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (BenMukebo) [inline README.md]: - [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

File: README.md [hunk 2]
```diff
@@ -1 +1,128 @@
...
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+
+# 📖 [BookStore] <a name="about-project"></a>
+
+BookStore
+
```
Comment (BenMukebo) [inline README.md]: - [OPTIONAL]  Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Header from './components/Header';
+import Books from './components/Books';
+import Categories from './components/Categories';
+
+function App() {
```
Comment (BenMukebo) [inline src/App.js]: - [ ] Great job so far 👏🏻 However I'm getting this error from the `/App.css';` file actually you're trying to import a file that does not exist in your project dir, please remove this line `import './App.css';` to fix the issue.

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Lawmsangi,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- The project was initialized using Create React App.  ✔️ 
- There are 2 `<Route>`s added: `/` and `/categories`
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @Lawmsangi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `Git` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**